SU2_CFD/src/solvers/CDiscAdjSolver.cpp

```diff
@@ -178,64 +151,12 @@ void CDiscAdjSolver::SetRecording(CGeometry* geometry, CConfig *config){
 
 }
 
-void CDiscAdjSolver::SetMesh_Recording(CGeometry** geometry, CVolumetricMovement *grid_movement, CConfig *config) {
-
-
-//  bool time_n_needed  = ((config->GetUnsteady_Simulation() == DT_STEPPING_1ST) ||
-//      (config->GetUnsteady_Simulation() == DT_STEPPING_2ND)),
-//  time_n1_needed = config->GetUnsteady_Simulation() == DT_STEPPING_2ND;
-
-//  unsigned long ExtIter = config->GetExtIter();
-
-  unsigned long iPoint;
-  unsigned short iDim;
-
-  /*--- Reset the solution to the initial (converged) position ---*/
```

Comment:
I am deleting dead code in the disc adj solvers as I move through this.
If you think I shouldn't speak now.

Reply:
If you put these changes into another pull request, they can be merged earlier.

Reply:
But that would have been smart... We don't do that here